src/Mono.Android/java/mono/android/Runtime.java

```diff
@@ -1,12 +1,25 @@
 package mono.android;
 
 public class Runtime {
+	static java.lang.Class java_lang_Class;
+	static java.lang.Class java_lang_System;
+	static java.lang.Class java_util_TimeZone;
+	static java.lang.Class mono_android_IGCUserPeer;
+	static java.lang.Class mono_android_GCUserPeer;
+
+	static {
```

Comment:
Is a `static` block actually needed, as opposed to just inline initialization?

Reply:
Is there a difference? I thought this looked nicely delimited from the other code, just that

Reply:
The difference is that when we add Yet Another Field, using "inline assignment" means the new field is a ~1 line change, not a  2 line change, and it would presumably be harder to accidentally forget the assignment line.

Otherwise, I don't think there's a huge difference.